packages/styleguide/src/components/App/App.js

```diff
@@ -164,37 +192,32 @@ const GlobalStyle = createGlobalStyle`
   *::after { box-sizing: border-box; }
   *::before { box-sizing: border-box; }
 `;
-/* eslint-enable */
 
-const PageLayout = styled.div``;
+const Overlay = styled('div')`
```

Comment:
Toto by som tiez vyhodil z App, ked je to spojene so Sidebarom ci?

Reply:
nieje to spojene s App, ale potrebujem to na inom mieste ako je Sidebar component

Reply:
No stale by som to vyhodil z App, ak to nie je spojne z App.